eng/pipelines/common/xplat-setup.yml

```diff
@@ -87,6 +87,10 @@ jobs:
         ${{ if eq(parameters.osGroup, 'OSX') }}:
           vmImage: 'macOS-10.14'
 
+        # Public OSX Build Pool
+        ${{ if eq(parameters.osGroup, 'iOS') }}:
+          vmImage: 'macOS-10.14'
```

Comment:
To avoid duplication and as I envision the iOS pool to always lineup to the OSX pool, why not do an or in the if above where we set the vmImage for OSX?

Reply:
I can envisage a time when this might need to differ - e.g. if we need to support a bleeding-edge iOS, then we'll need a bleeding edge XCode, which might need a new OS release (this can happen & has happened). We'll just end up splitting the conditional back out again as & when

Reply:
Ok, sounds good.